css/product.css

```diff
@@ -0,0 +1,150 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* navbar */
+.logo {
+  margin-right: var(--size-2);
+}
+
+.btn_logout {
+  padding: var(--size-3) var(--size-6);
+}
+
+.icon_badge .btn .badge {
+  height: var(--size-5);
+  width: var(--size-5);
+  top: -20%;
+  right: 0;
+}
+
+/* main */
+.main {
+  display: grid;
+}
+
+/* aside */
+.aside {
+  grid-column: 1/-1;
+  grid-row: 1/2;
+  padding: var(--size-2) var(--size-4);
+  border-bottom: 1px solid var(--gray-100);
+}
+.products {
+  grid-row: 2/3;
+  grid-column: 1/-1;
+}
+
+.product_list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: var(--size-6);
+  margin-bottom: var(--size-4);
+}
+
+.filter {
+  justify-content: space-between;
+}
```

Comment:
You can consider making a justify-between utility class for the same

Reply:
Roger that